src/scripts/main.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+async function getData(url) {
+    return await fetch(url)
+        .then(response => response.json());
+}
+
+async function createToDoList () {
+    const table = document.querySelector('#tbody');
+    const toDoList = [await getData('https://jsonplaceholder.typicode.com/todos'),
+    await getData('https://jsonplaceholder.typicode.com/users')];
+
+    return table.innerHTML = `
+      ${toDoList[0].map(item => `
+        <tr>
+          <td>${item.title}</td>
+          <td>
+            <a href="mailto:${toDoList[1][item.userId - 1].email}">${toDoList[1][item.userId -1].name}</a>
```

Comment:
ця штука працює, тільки тому що перші 10 постів належать 1 юзеру, якби вони були перемішані, то дані були б не правильні
```suggestion
            <a href="mailto:${toDoList[1][item.userId - 1].email}">
              ${toDoList[1][item.userId -1].name}
            </a>
```